README.md

```diff
@@ -90,6 +91,47 @@ windows_batch 'echo some env vars' do
 end
 ```
 
+### windows_dns
+Configures A and CNAME records in Windows DNS. This requires the DNSCMD to be installed, which is done by adding the DNS role to the server or installing the Remote Server Admin Tools.
+
+#### Actions
+- :create: creates/updates the DNS entry
```

Comment:
A :install action that simply calls windows_feature to install the dns server might solve the requirement that DNSCMD.exe be installed.